Education.cs

```diff
@@ -0,0 +1,45 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using MarsQA_1.Helpers;
+using NUnit.Framework;
+using OpenQA.Selenium;
+
+namespace MarsQA_1.Pages
+{
+    class Education
+    {
+        private static IWebElement Education_ => Driver.driver.FindElement(By.XPath("//A[text()='Education']"));
+
+        public static string title_name;
+        private static IWebElement DeleteEduPen => Driver.driver.FindElement(By.XPath("(//td[text()='" + title_name + "']//following::i[@class='remove icon'])[1]"));
+        private static IList<IWebElement> EduTableRow => Driver.driver.FindElements(By.XPath("(//table[@class='ui fixed table'])[3]/tbody/tr"));
+        public static void Delete_Edu()
+        {
+            Helpers.Driver.TurnOnWait();
+
+            Education_.Click();
+            ExcelLibHelper.PopulateInCollection(@"C:\Users\Anusree Rakesh\Desktop\Mars1\MarsQA-1\SpecflowTests\Data\Data.xlsx", "Education");

```

Comment:
in this task, you need to use data table on specflow to pass data, so you will need to remove this helper